Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -1,41 +1,19 @@
-# Eye Tracking Examples in MRTK
-This page covers how to get quickly started with using Eye Tracking in MRTK by building on our provided MRTK example package.
-The samples let you experience one of our new magical input capabilities: **Eye Tracking**! 
-The demo include a number of different use cases for seamlessly combining information about what you are looking at with **Voice** and **Hand** input. 
-This enables users to quickly and effortlessly select and move holograms across their view simply by looking at a target and saying _'Select'_ or performing a hand gesture. 
+# Eye tracking examples in MRTK
+This page covers how to get quickly started with using eye tracking in MRTK by building on our provided [MRTK eye tracking example package](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking).
```

Comment:
```suggestion
This page covers how to get quickly started with using eye tracking in MRTK by building on our provided [MRTK eye tracking examples](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking).
```

Reply:
Mainly because there's no separate package for eye tracking examples, they are part of the examples package.

Reply:
I don't want to lead people down a path of asking "where's the eye tracking package?"